src/errors.rs

```diff
@@ -86,7 +86,10 @@ pub enum RustupError {
     RunningCommand { name: OsString },
     #[error("toolchain '{0}' is not installable")]
     ToolchainNotInstallable(String),
-    #[error("toolchain '{0}' is not installed")]
+    #[error("toolchain '{0}' is not installed.{}", match .0.to_string().parse::<PartialToolchainDesc>() {
```

Comment:
1. `.to_string()` doesn't seem required here.
2. I prefer using `.map_or_default()` over `match`.

Reply:
I think `.to_string()` is required because it needs to cast `ToolchainName` to a `String`, or did I misunderstand something?

Reply:
> I think `.to_string()` is required because it needs to cast `ToolchainName` to a `String`, or did I misunderstand something?

@jtr860830 No, that's an oversight on my side, sorry. We're all good now.